src/main/java/com/app/odapi/service/OdApiTaskService.java

```diff
@@ -0,0 +1,217 @@
+package com.app.odapi.service;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import com.app.odapi.model.AirportsRequest;
+import com.app.odapi.model.AirportsResponse;
+import com.app.odapi.model.CountriesRequest;
+import com.app.odapi.model.CountriesRequestData;
+import com.app.odapi.model.CountriesResponse;
+import com.app.odapi.model.DestinationProperties;
+import com.app.odapi.model.OdApiResponse;
+import com.app.odapi.model.OriginsRequest;
+import com.app.odapi.model.RegionsRequest;
+import com.app.odapi.model.RegionsRequestData;
+import com.app.odapi.model.RegionsResponse;
+import com.app.odapi.model.RoutesRequest;
+import com.app.odapi.model.RoutesRequestData;
+import com.app.odapi.model.RoutesResponse;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Service
+public class OdApiTaskService {
+
+	@Value("${file.origins:test}")
+	private String originsFile;
+	@Value("${file.regions:test}")
+	private String regionsFile;
+	@Value("${file.countries:test}")
+	private String countriesFile;
+	@Value("${file.routes:test}")
+	private String routesFile;
+
+	public OdApiResponse getAllOdApiData() throws JsonParseException, JsonMappingException, IOException {
+		  ClassLoader classLoader = new OdApiTaskService().getClass().getClassLoader();
+		  ObjectMapper ob = new ObjectMapper();
+		  OdApiResponse response = new OdApiResponse();
+		  
+		  Date date = new Date();
+		  SimpleDateFormat DateFor = new SimpleDateFormat("MM/dd/yyyy hh:mm:ss aa");
+		  String stringDate = DateFor.format(date);
+           
+		  File originsfile = new File(classLoader.getResource(originsFile).getFile());
+		  OriginsRequest airportRequest = ob.readValue(originsfile,
```

Comment:
read from URL of the file instead of File, its fails in some machines
OriginsRequest airportRequest = ob.readValue(classLoader.getResource(originsFile), OriginsRequest.class);